app/controllers/jobs_controller.rb

```diff
@@ -5,5 +5,10 @@ def index
 
   def show
     @job = Job.find(params[:id])
+    @job_description = @job.description
+      .gsub("]", "")
+      .gsub("[", "")
+      .gsub("\"", "")
+      .gsub(",", "")
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.